README.rst

```diff
@@ -146,6 +150,16 @@ Conda (Anaconda)
 
 Currently, it supports only python version 2.7, 3.6 and 3.7.
 
+Snap (Snapcraft)
+----------------
+
+.. code-block:: bash
+
+   snap install dvc --classic
```

Comment:
what will it install without `--classic`? Latest tag, right?

Reply:
nothing will install without `--classic` (well, my initial builds may install with `--edge` and without `--classic` but they aren't usable)

`--classic` is a security thing (insists users are aware that confinement isn't `strict`). Don't see a way around it as the `git-ubuntu` snap also needs `--classic`.

Reply:
just updated the PR description ot make it more clear

Reply:
Oh, now I get it. Thanks for clarifying!

Reply:
@casperdcl Let's add a note that we are waiting for `classic` approval and let's list those flags that you need to provide to install it even without `classic` approved.

Reply:
trying to push an initially useful version without `--classic` instead now. Installation won't be trivial (instructions would need updating) otherwise